src/aiida_quantumespresso/workflows/protocols/xspectra/base.yaml

```diff
@@ -0,0 +1,46 @@
+default_inputs:
+    clean_workdir: True
+    kpoints_distance: 0.15
+    kpoints_force_parity: False
+    xspectra:
+        parameters:
+            INPUT_XSPECTRA:
+                calculation: xanes_dipole
+                xonly_plot: False
+                xniter: 2000
+                xcheck_conv: 10
+                xerror: 0.001
+            PLOT:
+                xgamma: 0.5
+                xnepoint: 2000
+                xemin: -10
+                xemax: 30
+                terminator: True
+                cut_occ_states: True
+            CUT_OCC:
+                cut_desmooth: 0.1
+        metadata:
+            options:
+                resources:
+                    num_machines: 1
+                max_wallclock_seconds: 43200  # Twelve hours
+                withmpi: True
+default_protocol: one_shot
```

Comment:
If `moderate` refers to, say, a "general-purpose" method suitable for any given system, then I think that would be reasonable to change `one_shot` to `moderate`. `production` in this case would be better defined as `fast`, since the aim is to speed up the Lanczos recursion and reduce the amount of time spent plotting the spectrum at the end - as evidenced by the fact that `production` only plots 5 points in total - which works well if it's being used for testing purposes or to re-plot the spectrum in a follow-up calculation. If we're sticking to the convention of `fast`, `moderate`, and `precise` then I could maybe add a `precise` method with tighter convergence criteria and shorter `kpoints_distance` parameter, if that would be welcomed? Keeping the `re_plot` protocol is a useful shortcut in any case, so I think it would be best to keep this as an additional protocol.

Reply:
Yeah, the idea is that `moderate` is the default and should give a calculation with reasonable precision while not being too expensive, a good middle ground. The `fast` protocol should be faster, and should typically be used for testing or if precision is not that important. The `precise` option is reserved for when you need really precise calculations for production, such as actual research. I agree that keeping `re_plot` as an additional protocol is useful.